<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(40, 44, 52);
border-radius: 5px;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(40, 44, 52);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>605</width>
        <height>50</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(40, 44, 52);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <widget class="QPushButton" name="btnClose">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>0</y>
         <width>55</width>
         <height>50</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="toolTip">
        <string extracomment="Close"/>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	border:none;
	border-radius: 0px;
}

QPushButton:hover{
	background-color: rgb(255, 0, 0);
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../resources/assert/x.svg</normaloff>../resources/assert/x.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Baskerville Old Face</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
padding-left:5px;</string>
       </property>
       <property name="text">
        <string>APIKEY</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>605</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(40, 44, 52);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <widget class="QLineEdit" name="apiKeyEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>451</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Baskerville Old Face</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
color: rgb(255, 255, 255);
padding-left: 5px;
border-radius: 5px;
border: 2px solid rgb(255,255,255);	
}
</string>
       </property>
       <property name="placeholderText">
        <string>Enter api key</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="btnSaveKey">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>90</y>
         <width>111</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Baskerville Old Face</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: rgb(66, 66, 66);
	border:none;
	border-radius: 5px;
}

QPushButton:hover{
	border-radius: 10px;
}

QPushButton:pressed{
	border: 2px solid rgb(255,255,255);	
}</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../resources/assert/key.svg</normaloff>../resources/assert/key.svg</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="keyStatus">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>581</width>
         <height>51</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Baskerville Old Face</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(66, 66, 66);
padding-left: 2px;
border-radius: 5px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
